packages/eslint-plugin/lib/config/typescript.ts

```diff
@@ -1,76 +1,77 @@
 import type { Linter } from "eslint";
-import { typescriptFiles } from "../utils/patterns";
+import tseslint from "typescript-eslint";
+import { typeScriptFiles } from "../utils/patterns.ts";
 
-const config: Linter.Config = {
-    overrides: [
-        {
-            files: typescriptFiles,
-            parser: "@typescript-eslint/parser",
-            plugins: ["@typescript-eslint"],
-            extends: [
-                "plugin:@typescript-eslint/eslint-recommended",
-                "plugin:@typescript-eslint/recommended"
-            ],
-            rules: {
-                // @typescript-eslint/recommended disables
-                "@typescript-eslint/no-non-null-assertion": "off",
+const config: Linter.FlatConfig[] = [
+    {
+        files: typeScriptFiles,
+        plugins: {
+            // @ts-ignore
+            "@typescript-eslint": tseslint.plugin
```

Comment:
So I guess that because of this plugin, you don't need to extends `plugin:@typescript-eslint/eslint-recommended` and `plugin:@typescript-eslint/recommended` anymore?

By extend, I mean using the spread syntax in the `rules` section.

Reply:
I will need to check on this. They changed their structure but there is still a recommended config.

Reply:
I've added the missing recommended rules